docs/aws/audit/elbmonitoring/rules/elb_app_tier_health_check.mdx

```diff
@@ -24,6 +24,210 @@ HIPAA
+   ```sh
+   aws elb configure-health-check --load-balancer-name <load-balancer-name> --health-check Target=HTTP:80/index.html,Interval=30,Timeout=5,UnhealthyThreshold=2,HealthyThreshold=2
+   ```
+
+3. **Verify Health Check Configuration:**
+   After configuring the health check, verify the settings to ensure they are correctly applied. Use the following command to describe the health check configuration of your ELB:
+   ```sh
+   aws elb describe-load-balancers --load-balancer-names <load-balancer-name>
+   ```
+
+4. **Monitor Health Check Status:**
+   Regularly monitor the health check status of your instances to ensure they are healthy and the configurations are working as expected. Use the following command to describe the instance health:
+   ```sh
+   aws elb describe-instance-health --load-balancer-name <load-balancer-name>
+   ```
+
+By following these steps, you can ensure that the right health check configurations are used for your App-Tier ELBs in AWS using the AWS CLI.
+</Accordion>
+
+<Accordion title='Using Python'>
+To prevent misconfigurations related to health check configurations for App-Tier Elastic Load Balancers (ELBs) in AWS using Python scripts, you can use the Boto3 library, which is the AWS SDK for Python. Here are the steps to ensure the right health check configurations are used:
+
+1. **Install Boto3 Library**:
+   Ensure you have the Boto3 library installed. If not, you can install it using pip:
+   ```bash
+   pip install boto3
+   ```
+
+2. **Set Up AWS Credentials**:
+   Make sure your AWS credentials are configured. You can set them up using the AWS CLI or by creating a `~/.aws/credentials` file.
+
+3. **Create a Python Script to Configure Health Checks**:
+   Write a Python script to configure the health checks for your ELBs. Below is an example script:
+
+   ```python
+   import boto3
+
+   # Initialize a session using Amazon EC2
+   session = boto3.Session(
+       aws_access_key_id='YOUR_ACCESS_KEY',
+       aws_secret_access_key='YOUR_SECRET_KEY',
+       region_name='YOUR_REGION'
+   )
+
+   # Initialize the ELB client
+   elb_client = session.client('elb')
+
+   # Function to configure health check for a given ELB
+   def configure_health_check(elb_name, target, interval, timeout, healthy_threshold, unhealthy_threshold):
+       response = elb_client.configure_health_check(
+           LoadBalancerName=elb_name,
+           HealthCheck={
+               'Target': target,
+               'Interval': interval,
+               'Timeout': timeout,
+               'UnhealthyThreshold': unhealthy_threshold,
+               'HealthyThreshold': healthy_threshold
+           }
+       )
+       return response
+
+   # Example usage
+   elb_name = 'your-elb-name'
+   target = 'HTTP:80/ping'  # Example target
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>